addons/packages/antrea/1.2.3/test/antrea_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 
 	corev1 "k8s.io/api/core/v1"
+	appsv1 "k8s.io/api/apps/v1"
```

Comment:
@blc1996 This file is failing the Golang linter check here (I believe these imports should be alphabetized). You can run the check yourself with `make lint` to see the error.

Reply:
Fixed